README.md

```diff
@@ -66,10 +66,22 @@ const response = await fetch(`${baseUrl}/guests/1`, { method: 'DELETE' });
 const deletedGuest = await response.json();
 ```
 
+<!--
+
 ## Deploying to Heroku
 
 Create a Heroku account at [Heroku - Sign up](https://signup.heroku.com/), and then click on this button:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/upleveled/express-guest-list-api-memory-data-store/tree/main)
 
 This will set up a new application on your Heroku account using this repo as a template.
+
+-->
+
+### Run on Replit
+
+Create a Replit account at [Replit - Sign up](https://replit.com/signup), and then click on this button:
+
+[![Run on Repl.it](https://repl.it/badge/github/unknownblueguy6/MineSweeper)](https://replit.com/github/upleveled/express-guest-list-api-memory-data-store)
```

Comment:
- [ ] the image is still pointing at the other repo
- [ ] Replit does not have a period in the name

Reply:
818df70